Update what is platform engineering page (#3136)
pulumi/pulumi-hugo: themes/default/content/what-is/what-is-platform-engineering.md

---
title: What is Platform Engineering?
allow_long_title: true
meta_desc: |
    Understand what platform engineering is, along with the main benefits and importance
    for modern application development.

type: what-is
page_title: What is Platform Engineering?

---

## What is platform engineering?

Platform engineering is a set of modern engineering practices where infrastructure teams build shared tools and services to help development teams develop, deploy, and operate cloud infrastructure on a self-service basis. This includes cloud infrastructure, container orchestration platforms, databases, networking, monitoring, code repositories, and deployment pipelines.

> Platform teams use a customer-driven mindset where they treat the applications developers that they serve as customers that must be understood and won over through products that solve their problems. The product is infrastructure tooling that development teams use to provision and manage standardized infrastructure for their applications and services. Typically, these tools have built-in guardrails that enforce best practices and security standards without impeding developers’ agility and workflow.

Reviewer: ```suggestion
Platform teams use a customer-driven mindset where they treat the application developers that they serve as customers that must be understood and won over through products that solve their problems. The products these teams offer are infrastructure tools and building blocks that development teams use to provision and manage standardized infrastructure for their applications and services. Typically, these tools have built-in guardrails that enforce best practices and security standards without impeding developers’ agility and workflow.
```

The responsibilities of a platform engineer may include the following:

> * Serving development teams with self-service infrastructure tooling and golden paths to production

Reviewer: ```suggestion
* Providing development teams with self-service infrastructure tooling and golden paths to production
```

* Defining and creating standardized cloud architectures used by development teams, typically using an [infrastructure as code framework](/what-is/what-is-infrastructure-as-code/)

* [Automating infrastructure and operational tasks](/automation/) using tools such as infrastructure as code and continuous integration/delivery (CI/CD) pipelines

* Improving internal infrastructure tooling and processes in order to improve developer productivity, increase velocity, and reduce errors and downtime

* Defining and enforcing best practices and security standards for infrastructure, such as by using a [policy as code framework](/crossguard)
Researching and evaluating new technologies and tools to continually improve internal tooling

## What is a platform engineer?

> Platform engineer is a term used to describe the engineers that make up a platform team. Typically, these engineers have multi-disciplinary skills, experience, and empathy needed to build a great product, serve developers’ needs, and “go to market” within their company. Often, they have experience with multiple engineering disciplines like infrastructure or DevOps and software engineering. The reality is that many engineers who perform platform engineering responsibilities do not have the title “platform engineer.” In practice they have varying job titles like software engineer, DevOps engineer, SRE, cloud architect, cloud engineer, and more. By providing developers with infrastructure and tooling to deploy and operate their applications efficiently, platform engineers enable developers to focus on building great software.

Reviewer: ```suggestion
Platform engineer is a term used to describe the engineers that make up a platform team. Typically, these engineers have the multi-disciplinary skills, experience, and empathy needed to build a great product, serve developers’ needs, and “go to market” within their company. Often, they have experience with multiple engineering disciplines like infrastructure or DevOps and software engineering. The reality is that many engineers who perform platform engineering responsibilities do not have the title “platform engineer.” In practice they have varying job titles like software engineer, DevOps engineer, SRE, cloud architect, cloud engineer, and more. By providing developers with infrastructure and tooling to deploy and operate their applications efficiently, platform engineers enable developers to focus on building great software.
```
